lib/components/parallax_component.dart

```diff
@@ -93,10 +94,12 @@ class ParallaxLayer {
     // Size of the area to paint the images on
     final Vector2 paintSize = count..multiply(_imageSize);
     _paintArea = paintSize.toRect();
+
+    sized = true;
   }
 
   void update(Vector2 delta) {
-    if (!loaded()) {
+    if (!sized) {
```

Comment:
what was wrong with the loaded() method?

Reply:
See https://github.com/flame-engine/flame/pull/549#discussion_r550809473 and the corresponding changelog entry